specification/ekg/resource-manager/Microsoft.EKG/preview/2018-12-03/ekgSwagger.json

```diff
@@ -0,0 +1,586 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Enterprise Knowledge Graph Service",
+    "description": "Azure Enterprise Knowledge Graph Service is a platform for creating knowledge graphs at scale.",
+    "version": "2018-12-03"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EKG/ekgService/{resourceName}": {
+      "put": {
```

Comment:
As this is a tracked resource, it requires a PATCH api that supports update of tags. See the [RPC](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#patch-resource)